.Lib9c.Tests/Model/State/SheetStateTest.cs

```diff
@@ -0,0 +1,37 @@
+namespace Lib9c.Tests.Model.State
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using Libplanet;
+    using Nekoyume;
+    using Nekoyume.TableData;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class SheetStateTest
+    {
+        private readonly ITestOutputHelper _testOutputHelper;
+
+        public SheetStateTest(ITestOutputHelper testOutputHelper)
+        {
+            _testOutputHelper = testOutputHelper;
+        }
+
+        [Fact]
+        public void PrintSheetAddresses()
+        {
+            Assembly assembly = Assembly.GetAssembly(typeof(ISheet));
+            Assert.NotNull(assembly);
+
+            IEnumerable<string> sheetNames = assembly.GetTypes()
+                .Where(type => type.IsClass && !type.IsAbstract && type.Name.EndsWith("Sheet"))
```

Comment:
Of course, I know it uses `Sheet` suffix for `Sheet<T>` implementations now, how about using [`Type.IsAssignableFrom(Type)`](https://docs.microsoft.com/en-us/dotnet/api/system.type.isassignablefrom?view=netcore-3.1)? Like...

```suggestion
                .Where(type => type.IsClass && !type.IsAbstract && typeof(ISheet).IsSubclassOf(type))
```

It is just my suggestion, I think you can disagree with it 😀

Reply:
I tried like below but output is empty. I found [this](https://docs.microsoft.com/en-us/dotnet/api/system.type.issubclassof?view=net-5.0#remarks).

```suggestion
                .Where(type => type.IsClass && !type.IsAbstract && typeof(ISheet).IsSubclassOf(type))
```

And I tried like below and succeed.

```suggestion
                .Where(type => type.IsClass && !type.IsAbstract && typeof(ISheet).IsAssignableOf(type))
```

Maybe there is a typo in your suggestion.
Thank you!